src/main/events/catalogue/search-games.ts

```diff
@@ -1,12 +1,19 @@
 import { registerEvent } from "../register-event";
 import { searchGames } from "../helpers/search-games";
-import { CatalogueEntry } from "@types";
+import { CatalogueEntry, PaginationArgs } from "@types";
+import { defaults } from "lodash-es";
+
+const DEFAULT_SEARCH_ARGS: PaginationArgs = {
+  take: 12,
+  skip: 0,
```

Comment:
instead of skip, you could also just pass the page as a default for zero, and calculate the skip inside the function itself, then you dont need to know the default value for take when you just wannt to go to the page like 5 using the default "take" parameter

Reply:
Actually your suggestion was exactly the previous version of the function when i modified it initially for this pagination feature, but I thought that it is more reusable like this? Like if in the future we had to use this function to display different number of results we will able to do this without revising the code again and modify it

Reply:
+/-, like, how it is now, you need to know the `take` parameter to send the `skip` correctly considering the number of the `page` you want, so you basically dont make use of the default value for `take`, as you need to consider it anyway to send the `skip`.

if you send only `take` and `page`, both optional to 12 an 0 respectively, you dont need to care about any of this logic, and can even just send the `page` arg for any request. and you also has access to the `take` (default or not) used on the query to see if you are on the last page (query could also return a parameter with the number of pages tho)

what do you think @zamitto?

Reply:
I kinda agree with @piradata on this one, but the default to 12 on `take` I think that should be a global default value, indeed of having one here, another one in `src/renderer/src/pages/home/search-results.tsx` and so on.